Source/common/santa.proto

```diff
@@ -552,6 +552,11 @@ message FileAccess {
     POLICY_DECISION_ALLOWED_AUDIT_ONLY = 3;
   }
   optional PolicyDecision policy_decision = 6;
+
+  // Used to link a single operation emitting multiple FileAccess messages.
+  // This can happen, for example, when a single operation violates both Data
+  // and Process File Access Authorization rules.
+  optional string fingerprint = 7;
```

Comment:
If you're still looking for alternative names for `fingerprint`, how about `operation_id`?

Reply:
Ooo I think i like this name better, it's more self-describing. Will update.